app/app.py

```diff
@@ -11,4 +11,4 @@ def hello_world():
 
 
 if __name__ == "__main__":
-    app.run(port=os.environ.get("PORT", 3000), host="0.0.0.0")
+    app.run(port=os.environ.get("PORT", 5000), host="0.0.0.0")
```

Comment:
No need to change the code. You can either pass `-e PORT=5000` to the `docker run` command, or simply map it like `docker run -p 5000:3000 image_name:tag`